.travis.yml

```diff
@@ -104,29 +103,45 @@ jobs:
 
     - stage: quicktest
       php: 5.4
+      dist: trusty
```

Comment:
Oh, sneaky. I like this trick!

Reply:
*grin* I thought you knew this one - IIRC we used it in WPCS for `precise` when we still needed to support PHP 5.2/5.3.

Reply:
I think I have seen it before, but I hadn't remembered it. I can now change VIPCS back again to use a newer dist overall, but just add trusty for 5.4 and 5.5 envs :-)

Reply:
I actually got that commit ready locally already ;-)